common/common.h

```diff
@@ -138,6 +138,7 @@ template <class> inline constexpr bool always_false_v = false;
 } // namespace sorbet
 
 std::string demangle(const char *mangled);
+std::string exec(std::string cmd);
```

Comment:
We use a C++ library called `subprocess` (inspired by the python library of the same name, and similar to the library we use in Stripe's codebase). Can you use that instead of raw `exec`?

Reply:
You can grep the source code to see examples of its use